src/Ledger/Utxo.lagda

```diff
@@ -449,7 +450,7 @@ data _⊢_⇀⦇_,UTXO⦈_ where
     in
     ∙ txins ≢ ∅                              ∙ txins ∪ refInputs ⊆ dom utxo
     ∙ txins ∩ refInputs ≡ ∅                  ∙ inInterval slot txvldt
-    ∙ feesOK pp tx utxo ≡ true               ∙ consumed pp s txb ≡ produced pp s txb
+    ∙ feesOK pp _ tx utxo ≡ true             ∙ consumed pp s txb ≡ produced pp s txb
```

Comment:
New problem: we need `paramsWellFounded pp` here.  How can we incorporate that condition into the `UTXO` STS?